### PR TITLE
Update how-to-join-content-from-dissimilar-files-linq.md

### LINKED ISSUE #8841: The output is not as described
The output I get begins with
Merge two spreadsheets:
Omelchenko,Svetlana,111, 97, 92, 81
O'Donnell,Claire,112, 75, 84, 91
and should begin with
Merge two spreadsheets:
Omelchenko, 97, 92, 81, 60
O'Donnell, 75, 84, 91, 39
.
I conclude that the firstname and the student ID have been added and that the grades for the fourth exam are missing.

To my opininon the lines
    Select n(0) & "," & n(1) & "," & n2(0) & "," & n2(1) & "," &  
      n2(2) & "," & n2(3)  
need to be replaced by the lines
    Select n(0) & "," & n2(1) & "," & n2(2) & "," &
      n2(3) & "," & n2(4)
in order to get the described output.

Greets, Evert Kuijpers from Tilburg in The Netherlands

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 776aadf4-4af5-753d-8ce9-ca5dbe3681f4
* Version Independent ID: 4d9cb98b-c8a7-6e62-d0a0-a3247c627347
* Content: [How to: Join Content from Dissimilar Files (LINQ) (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq)
* Content Source: [docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md](https://github.com/dotnet/docs/blob/master/docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md)
* Product: **dotnet-visualbasic**
* GitHub Login: @KathleenDollard
* Microsoft Alias: **kdollard**

### PR DESCRIPTION
**Pull request by KathleenDollard**
In response to https://github.com/dotnet/docs/issues/8841

## Summary

Per issue, changed code to not output the ID, as it is not in the displayed output 

Fixes #8841

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Class JoinStrings
                             From id In scores   
                             Let n2 = id.Split(New Char() {","})   
                             Where Convert.ToInt32(n(2)) = Convert.ToInt32(n2(0))
-                            Select n(0) & "," & n(1) & "," & n2(0) & "," & n2(1) & "," &  
+                            Select n(0) & "," & n2(0) & "," & n2(1) & "," &
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md]: This is incorrect. It should be:
```
                            Select n(0) & "," & n2(1) & "," & n2(2) & "," &  
                              n2(3) & "," & n2(4)  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for addressing this issue, @KathleenDollard. Your change, though, returns the student ID, which isn't included in the output.

**Comment by rpetrusha** [comment]
Also, @KathleenDollard see my comments on the path in #8888

File: docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md (hunk 1 above)
Comment (mairaw) [reply docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md]:
```suggestion
                            Select n(0) & "," & n2(1) & "," & n2(2) & "," &  
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Class JoinStrings
                             From id In scores   
                             Let n2 = id.Split(New Char() {","})   
                             Where Convert.ToInt32(n(2)) = Convert.ToInt32(n2(0))
-                            Select n(0) & "," & n(1) & "," & n2(0) & "," & n2(1) & "," &  
+                            Select n(0) & "," & n2(0) & "," & n2(1) & "," &  
                               n2(2) & "," & n2(3)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md]:
```suggestion
                               n2(3) & "," & n2(4)  
```

**Review by mairaw — COMMENTED** [review]
Just put it @rpetrusha comments in the form of suggestions so you can easily batch them and commit.

**Pull request closed without merging**